multigeiger/version.h

```diff
@@ -1,15 +1,18 @@
-// Revision history:
+// To fit in 16bit for lora version number we have
+// some limits here: max. VERSION_MAJOR = 15, max. VERSION_PATCH = 15
+#define VERSION_MAJOR 1
+#define VERSION_MINOR 11
+#define VERSION_PATCH 0
+#define VERSION_DATE "2019-12-16"
```

Comment:
We could have a comment above this that this is a date in YYYY-MM-DD format.

It is clear with the current value, but not for every value.